scripts/migrations/20210915152208786_add_email_to_contacts.cr

```diff
@@ -0,0 +1,15 @@
+class AddEmailToContacts < Jennifer::Migration::Base
+  def up
+    change_table :contacts do |t|
```

Comment:
we already have uniqueness indexes for `Contact#description` and `Address#street` so let's avoid this migration

Reply:
I added this to add a test case for passing a value to the `unique_fields` array (see https://github.com/imdrasil/jennifer.cr/pull/389/files#diff-92581025c55dc5c29acaa36d1fbe56559b5a4157b9a25ee88c7dcfd0fbeaa643R302-R314). With only one unique field per model, we cannot test this, because the behavior of `#upsert()`, `#upsert([] of String)` and `#upsert(["unique_field_name"]` is the same. It only makes a difference if there is another constraint that can fail, right?